src/js/background.js

```diff
@@ -766,6 +789,51 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  showCookies: function(fqdn) {
+    let domain = window.getBaseDomain(fqdn);
+    chrome.cookies.getAll({domain}, cookies => {
+      console.log('Cookies for domain: ' + domain);
+      cookies.forEach(cookie => {
+        console.log('\t -------------');
+        console.log('\t cookie name: ' + cookie.name);
+        console.log('\t cookie domain: ' + cookie.domain);
+      });
+    });
+  },
+
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, tabs => {
+      tabs.forEach(tab => {
```

Comment:
Yeah, I agree. This is getting complicated enough, and isn't strictly "bacground" so I think it should go in its own file at this point.